cli/bump.go

```diff
@@ -5,22 +5,24 @@ import (
 	"strings"
 	"time"
 
+	"github.com/coder/coder/coderd/util/tz"
+
 	"github.com/spf13/cobra"
 	"golang.org/x/xerrors"
 
 	"github.com/coder/coder/codersdk"
 )
 
 const (
-	bumpDescriptionLong = `To extend the autostop deadline for a workspace.`
+	bumpDescriptionLong = `Specify a duration from now when you would like your workspace to shut down.`
 )
 
 func bump() *cobra.Command {
 	bumpCmd := &cobra.Command{
 		Args:        cobra.RangeArgs(1, 2),
 		Annotations: workspaceCommand,
-		Use:         "bump <workspace-name> <duration>",
-		Short:       "Extend the autostop deadline for a workspace.",
+		Use:         "bump <workspace-name> <duration from now>",
+		Short:       "Specify a duration from now when you would like your workspace to shut down.",
 		Long:        bumpDescriptionLong,
```

Comment:
You're using the same description for both the short and long description fields

Reply:
Embiggened one and smallified the other.